components/Report/Chart/ReportBarChart.tsx

```diff
@@ -0,0 +1,45 @@
+'use client';
+import { Bar, BarChart, CartesianGrid, Tooltip, XAxis, YAxis } from 'recharts';
+import styles from '../Report.module.css';
+
+type ReportBarChartProps = {
+    title: string;
+    data: Array<{ name: string; value: number; fill: string }>;
+};
+
+export default function ReportBarChart(props: ReportBarChartProps) {
+    const { title, data } = props;
+    return (
+        <div className={styles.outerBarChartContainer}>
+            <h2>{title}</h2>
+            <div className={styles.innerBarChartContainer}>
+                <div>
+                    {data.map((dataPoint: any) => (
```

Comment:
Similar to the pie chart we could consider removing values of 0 from being plotted. I'm less sure of this one though as having consistent x axes and blank spaces for categories that do have a value of 0 does make sense in a bar chart.